table/block_based/block_based_table_reader.h

```diff
@@ -65,9 +65,6 @@ class BlockBasedTable : public TableReader {
 
   // All the below fields control iterator readahead
   static const size_t kInitAutoReadaheadSize = 8 * 1024;
-  // Found that 256 KB readahead size provides the best performance, based on
-  // experiments, for auto readahead. Experiment data is in PR #3282.
-  static const size_t kMaxAutoReadaheadSize;
   static const int kMinNumFileReadsToStartAutoReadahead = 2;
```

Comment:
Is there a reason that you want to configure the MaxAutoReadaheadSize but leave the MinNumFilesToStartAutoReadahead and InitAutoReadaheadSize as constant?

Reply:
It was a requirement from a service owner to configure kMaxAutoReadaheadSize so that they disable prefetching for L0 files and enable it for other level files by configuring kMaxAutoReadaheadSize. Users can specify initial readahead_size in ReadOptions. kInitAutoReadaheadSize is used if user didn't specify  and after few seqeuntial reads RocksDB enable automatic prefetching.